
base error
==========

> cat_line(base)
Error in h() : foo
Calls: f -> g -> h
Run `rlang::last_error()` to see where the error occurred.
<error/rlang_error>
foo
Backtrace:
 1. global::f()
 2. global::g()
 3. global::h()
Run `rlang::last_trace()` to see the full context.
<error/rlang_error>
foo
Backtrace:
    █
 1. └─global::f()
 2.   └─global::g()
 3.     └─global::h()


rlang error
===========

> cat_line(rlang)
Error: foo
Run `rlang::last_error()` to see where the error occurred.
Run `rlang::last_error()` to see where the error occurred.
█
├─<error/rlang_error>
│ foo
│ Run `rlang::last_error()` to see where the error occurred.
└─<error/rlang_error>
  foo
  Run `rlang::last_error()` to see where the error occurred.
Backtrace:
 1. global::f()
 2. global::g()
 3. global::h()
 4. rlang::abort("foo")
 5. rlang:::signal_abort(cnd)
Run `rlang::last_trace()` to see the full context.
█
├─<error/rlang_error>
│ foo
│ Run `rlang::last_error()` to see where the error occurred.
└─<error/rlang_error>
  foo
  Run `rlang::last_error()` to see where the error occurred.
Backtrace:
    █
 1. └─global::f()
 2.   └─global::g()
 3.     └─global::h()
 4.       └─rlang::abort("foo")
 5.         └─rlang:::signal_abort(cnd)

